apps/frontend/src/app/services/data/local-user.service.ts

```diff
@@ -62,17 +64,20 @@ export class LocalUserService {
     return this.authService.isAuthenticated();
   }
 
```

Comment:
My suggestion is to create a `BehaviourSubject<boolean>` here that emits whenever the user logs in or out. There's several places in your changes that I think this would be useful, see other comments